server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfile.java

```diff
@@ -343,6 +341,13 @@ public void setExcluded(@Nullable Boolean excluded) {
         this.excluded = excluded;
     }
 
+    @Transient
```

Comment:
Does this need to be marked as `@Transient`? I wouldn't think so since it's not a field, getter, or setter...